C++/4. Advanced-Operators.md

```diff
@@ -2,11 +2,11 @@
 
 ## Start
 
-In this file I am going to teach you about advanced operators. There are three more types we are going to learn about - Assignment Operators, Relational Operators, and Logical Operators.
+In this file I am going to teach you about more advanced operators. There are three more types we are going to learn about - Compound-assignment operators, relational operators, and logical operators.
 
-## Assignment Operators
+## Coumpound-assignment Operators
 
-The Assignment Operators are basically shortcuts for making a variable be equal to a number (operator) itself:
+The Coumpound-assignment operators are basically shortcuts for assigning a variable with the result of applying an operator to itself and another value:
```

Comment:
doesn't make sense

Reply:
Apply an operator to `self` and an operand, then assign to `self`,
```c++
self = self {op} operand;

self {op}= operand;
```

How can I make this clearer, while being generic across operators? What part of it confuses you?

Reply:
the wording is confusing. AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA @darkdarcool you have any suggestions for the wording?

If not it is ok as it is pretty much understood below.

Reply:
Updated, what do you think about it now?